hal/network/ncp_client/sara/sara_ncp_client.cpp

```diff
@@ -1702,7 +1715,11 @@ int SaraNcpClient::initMuxer() {
     // Initialize muxer
     muxer_.setStream(serial_.get());
     muxer_.setMaxFrameSize(UBLOX_NCP_MAX_MUXER_FRAME_SIZE);
-    muxer_.setKeepAlivePeriod(UBLOX_NCP_KEEPALIVE_PERIOD);
+    if (ncpId() == PLATFORM_NCP_SARA_R510 && connState_ == NcpConnectionState::DISCONNECTED) {
+        muxer_.setKeepAlivePeriod(UBLOX_NCP_KEEPALIVE_PERIOD_R510);
```

Comment:
This ensures the keep alive is disabled on init in general.  If you just turn on cellular, but do not connect, it will be DISCONNECTED.  In that case we still want it to drop into low power mode.